src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -1003,16 +1006,6 @@ private SortedSet<MavenCoordinates> coreVersionFromWAR(UpdateSite.Data data) {
         return result;
     }
 
-    private UpdateSite.Data newUpdateSiteData(UpdateSite us, JSONObject jsonO) throws RuntimeException {
-        try {
-            Constructor<UpdateSite.Data> dataConstructor = UpdateSite.Data.class.getDeclaredConstructor(UpdateSite.class, JSONObject.class);
-            dataConstructor.setAccessible(true);
-            return dataConstructor.newInstance(us, jsonO);
-        }catch(Exception e){
-            throw new RuntimeException("UpdateSite.Data instantiation problems", e);
-        }
-    }
-    
```

Comment:
This reflection-based code apparently wasn't working properly anyway according to IntelliJ's static analysis checks.